doc/nrf/libraries/bin/lwm2m_carrier/index.rst

```diff
@@ -9,15 +9,18 @@ LwM2M carrier
 
 Several mobile network carriers specify their own LwM2M device management.
 The LwM2M carrier library facilitates an nRF91 device to connect to the device management servers, regardless of the other actions of the user application.
+For more information, see the list of `applicable carriers <Mobile network operator certifications_>`_, and the ref:`lwm2m_certification` subpage.
 
 .. important::
 
-   It is *mandatory* to include the LwM2M carrier library in any AT&T, T-Mobile, or Verizon certified products.
-
-For the `applicable carriers <Mobile network operator certifications_>`_, it is mandatory to include this library in your application for the certification to your end product.
+   It is *mandatory* to include the LwM2M carrier library in any Verizon certified product.
```

Comment:
Shouldn't we also mention that it is optional to include the library in any AT&T and T-Mobile certified product?